lib/format_dpdk.c

```diff
@@ -483,7 +509,7 @@ static inline int dpdk_init_environment(char * uridata, struct dpdk_format_data_
 		 * automatically but it's hard to tell that this is secondary
 		 * before running rte_eal_init(...). Currently we are limited to 1
 		 * instance per core due to the way memory is allocated. */
-		if (parse_pciaddr(uridata, &use_addr, &my_cpu) != 0) {
+		if (parse_pciaddr(uridata, &use_addr, &my_cpu, format_data) != 0) {
```

Comment:
Only applies to pci devices not the virtual DPDK devices. However, will be a non-issue once we move core pinning into the main libtrace library.